nixos/modules/config/system-path.nix

```diff
@@ -74,29 +63,6 @@ in
         '';
       };
 
-      defaultPackages = mkOption {
```

Comment:
Due to this removal of the option, `mkRemovedOptionModule` should added.

Reply:
Done, thanks! Is the message enough as is, or should there be more details?